src/graph.h

```diff
@@ -14,15 +14,21 @@ class Graph : public torch::CustomClassHolder {
   void LoadCSC(torch::Tensor indptr, torch::Tensor indices);
   void SetCSC(std::shared_ptr<CSC> csc);
   std::shared_ptr<CSC> GetCSC();
+  torch::Tensor GetcolIds();
+  void SetColIds(torch::Tensor column_ids);
   c10::intrusive_ptr<Graph> ColumnwiseSlicing(torch::Tensor column_ids);
   c10::intrusive_ptr<Graph> ColumnwiseSampling(int64_t fanout, bool replace);
   c10::intrusive_ptr<Graph> ColumnwiseFusedSlicingAndSampling(torch::Tensor column_ids, int64_t fanout, bool replace);
   torch::Tensor RowIndices();
+  torch::Tensor AllIndices();
+  std::tuple<torch::Tensor, torch::Tensor, torch::Tensor> Relabel();
```

Comment:
Could you give some explanations about what does this interface do? What are the three returned tensors?